third_party/move/move-model/src/builder/builtins.rs

```diff
@@ -0,0 +1,528 @@
+// Copyright (c) The Diem Core Contributors
```

Comment:
Unfortunately, even though was `git mv`ed from `spec_builtins.rs`, this history is lost after changes.